modules/layers/src/path-layer/path-layer.js

```diff
@@ -147,6 +148,7 @@ export default class PathLayer extends Layer {
         loop: props._pathType === 'loop',
         getGeometry: props.getPath,
         positionFormat: props.positionFormat,
+        resolution: this.context.viewport.resolution,
```

Comment:
As mentioned above, this will be an issue. For an experimental feature this is OK, but needs to moved out of the viewport. We can ideate on how to handle this (global flag, layer can generated multiple geometries, presence of globe view triggers a global flag, etc) but at least add TODO comments here.